mobile/src/main/java/ch/epfl/sdp/mobile/ui/game/GameScreen.kt

```diff
@@ -123,10 +123,17 @@ private val Message.text: String
  * @param modifier modifier the [Modifier] for this composable.
  */
 @Composable
-private fun Moves(
+fun Moves(
     moves: List<Move>,
     modifier: Modifier = Modifier,
+    firstColor: ChessBoardState.Color = White,
```

Comment:
Missing docu

Reply:
Will be resolved in another PR (Issue #344)